src/dashboards/sm-dns.json

```diff
@@ -84,11 +84,7 @@
     {
       "circleMaxSize": "10",
       "circleMinSize": "2",
-      "colors": [
-        "#37872D",
-        "#FA6400",
-        "#C4162A"
-      ],
+      "colors": ["#37872D", "#FA6400", "#C4162A"],
```

Comment:
[nit] Personally I'd split this commit into two commits, or even better, 2 PRs. That way the git-blame will be cleaner and easier to interact with in future, and code-review is easier too.

Why have you made this change, and is it in scope of the review? Of course I have no objections to this change, but I hope we won't see these lines of code flip-flopping back and forth between PRs.

Reply:
This was just the project json formatter having a field day, I think that this file had never been run through it before. Good points!